.gitignore

```diff
@@ -11,3 +11,4 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+test.http
```

Comment:
I don't think I need this file. Do I?🤔

Reply:
You don't. Unless you want to test the API in your IDE are using Rest Client.